### DISCUSSION TITLE
Plugin not starting with vscodium 1.8.5

### DISCUSSION THREAD

**Discussion by noaxie** [microsoft/vscode-cpptools#12116] [Q&A] [answered]
It seems that the plugin doesn't activate with vscodium 1.85.1 and remote ssh.

Trying to run c:c++ log diagnostics results in no output even with log level set to debug. 

Same configuration but with vscodium 1.72.2 and remote ssh works perfectly fine. 

Anyone else observed this behaviour?

Using version c/c++ 1.19.8

**Comment by sean-mcmanus** [comment, marked as answer]
@noaxie vscodium isn't supported and not allowed by the license as well, see https://github.com/microsoft/vscode-cpptools/issues/5980
